api/config/config_local.json

```diff
@@ -46,9 +46,9 @@
     "operationalTerraWorkspaceName": "aouwgscohortextraction",
     "operationalTerraWorkspaceBucket": "fc-56d2f6f5-3efa-46f7-8c01-0911fd77f888",
     "extractionMethodConfigurationNamespace": "aouwgscohortextraction-test",
-    "extractionMethodConfigurationName": "GvsExtractCohortFromSampleNames",
-    "extractionMethodConfigurationVersion": 15,
-    "extractionMethodLogicalVersion": 3,
+    "extractionMethodConfigurationName": "joelproto",
```

Comment:
This is a copy of https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl taken today - looks like commit ae8b6018ee4242d2cdd35da4fd318369a88ddef3